api/tests/core/search/test_serialize_algolia.py

```diff
@@ -88,6 +88,10 @@ def test_serialize_offer():
             "thumbUrl": None,
             "tags": [],
             "times": [],
+            "GTL_level1": None,
+            "GTL_level2": None,
+            "GTL_level3": None,
+            "GTL_level4": None,
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏
```suggestion
            "gtl_level4": None,